pyproject.toml

```diff
@@ -4,12 +4,7 @@ version = "0.1.0"
 description = "Tools for genome annotation"
 authors = ["Ensembl <dev@ensembl.org>"]
 maintainers = [
-    "Carlos Garcia Giron <dev@ensembl.org>",
-    "Leanne Haggerty <dev@ensembl.org>",
-    "Thibaut Hourlier <dev@ensembl.org>",
-    "Denye Ogeh <dev@ensembl.org>",
-    "William Stark <dev@ensembl.org>",
-    "Francesca Tricomi <dev@ensembl.org>",
+    "<dev@ensembl.org>",
```

Comment:
poetry is complaining during building 
Invalid maintainer string. Must be in the format: John Smith <john@example.com>
The command "poetry install" failed and exited with 1 during .
Could you please add` "ensembl <dev@ensembl.org>",`? it should be fine

Reply:
Sorry I missed this!